Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/MergerWindow.xaml

```diff
@@ -0,0 +1,55 @@
+﻿<Page x:Class="Ginger.ApplicationModelsLib.APIModels.APIModelWizard.MergerWindow"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+      xmlns:local="clr-namespace:Ginger.ApplicationModelsLib.APIModels.APIModelWizard"
+      xmlns:ginger="clr-namespace:Ginger"
+      mc:Ignorable="d" 
+      d:DesignHeight="780" d:DesignWidth="960" 
+      Title="API Compare And Merge" >
+
+
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="30" />
+            <RowDefinition Height="500*" />
+            <RowDefinition Height="30" x:Name="xMergerTextRow" />
+            <RowDefinition Height="400*" x:Name="xMergerAPIRow" />
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="300*" />
+            <ColumnDefinition Width="300*" />
+        </Grid.ColumnDefinitions>
+
+        <TextBlock Grid.Row="0" Grid.Column="0" Background="{StaticResource $BackgroundColor_DarkBlue}"
+                           Foreground="White" VerticalAlignment="Center" TextAlignment="Center" Text="Existing API" />
+        <Frame Grid.Row="1" Grid.Column="0" x:Name="xExistingAPIFrame" ScrollViewer.CanContentScroll="False" />
+
+        <GridSplitter x:Name="xComparerSplitter" ResizeDirection="Columns" Grid.Row="0" Grid.RowSpan="2" Grid.Column="1" Height="Auto" Width="5" HorizontalAlignment="Right" VerticalAlignment="Stretch" Background="{StaticResource $BackgroundColor_DarkBlue}" IsEnabled="True" />
+
+        <TextBlock Grid.Row="0" Grid.Column="1" Background="{StaticResource $BackgroundColor_DarkBlue}"
+                           Foreground="White" VerticalAlignment="Center" TextAlignment="Center" Text="Learned API" />
+        <Frame Grid.Row="1" Grid.Column="1" x:Name="xLearnedAPIFrame" ScrollViewer.CanContentScroll="False"/>
+
+        <Grid x:Name="xBaseSelectionSection" Grid.Row="2" Grid.Column="0" Grid.ColumnSpan="2" Background="{StaticResource $BackgroundColor_Gray}">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="160" />
+                <ColumnDefinition Width="160" />
+                <ColumnDefinition Width="160" />
+            </Grid.ColumnDefinitions>
+
+            <TextBlock Text="Base API Model : " Grid.Column="0" VerticalAlignment="Center" FontWeight="Bold" FontSize="20"/>
+            <Button x:Name="xMatchedAPIAsBase" Width="120" Content="Matching API" Click="XMatchedAPIAsBase_Click" Grid.Column="1" Style="{StaticResource $InputButtonStyle}" />
```

Comment:
controls names must include the control type- like xMatchedAPIAsBase -> xMatchedAPIAsBaseBtn
please handle for all needed controls in this page

Reply:
Done !